test/e2e/k8s_util.go

```diff
@@ -941,7 +941,8 @@ func (k *KubernetesUtils) Cleanup(namespaces []string) {
 
 	for _, ns := range namespaces {
 		log.Infof("Deleting test Namespace %s", ns)
-		if err := k.clientset.CoreV1().Namespaces().Delete(context.TODO(), ns, metav1.DeleteOptions{}); err != nil {
+		// Deleting the namespace resource from the API
```

Comment:
The commen is not useful as it doesn't add more information than the function name and the above log.